packages/client/src/config.ts

```diff
@@ -167,6 +167,7 @@ function initWitnesses(options: SignOptions): Witness[] {
     witnesses.push(
       new RekorWitness({
         rekorBaseURL: options.rekorURL,
+        entryType: 'intoto',
```

Comment:
Maintaining compatibility in the `client` package my explicitly requesting the "intoto" entry type.